From b3f69e3db7838c4536699a729a38a46ef6a98a8b Mon Sep 17 00:00:00 2001
From: Mikhail Kasimov <mikhail.kasimov@gmail.com>
Date: Thu, 10 May 2018 06:15:55 +0300
Subject: [PATCH] man: systemd-escape: add missed short keys (#8944)

Added short keys -u and -m for --unescape and --mangle respectively. These short keys are present in systemd-escape --help output and are absent in man systemd-escape page.

(cherry picked from commit 1a17bac57ade0745525b1ba592c3acc973b1ed74)
---
 man/systemd-escape.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/man/systemd-escape.xml b/man/systemd-escape.xml
index 513e9a6a97..98cedef77d 100644
--- a/man/systemd-escape.xml
+++ b/man/systemd-escape.xml
@@ -122,6 +122,7 @@
 
       <varlistentry>
         <term><option>--unescape</option></term>
+        <term><option>-u</option></term>
 
         <listitem><para>Instead of escaping the specified strings,
         undo the escaping, reversing the operation. May not be used in
@@ -132,6 +133,7 @@
 
       <varlistentry>
         <term><option>--mangle</option></term>
+        <term><option>-m</option></term>
 
         <listitem><para>Like <option>--escape</option>, but only
         escape characters that are obviously not escaped yet, and
-- 
2.17.0

